Tools/wasm/README.md

```diff
@@ -9,68 +9,19 @@ compilation of CPython to WebAssembly (WASM). Python supports Emscripten
 run in modern browsers and JavaScript runtimes like *Node.js*. WASI builds
 use WASM runtimes such as *wasmtime*.
 
-Users and developers are encouraged to use the script
-`Tools/wasm/wasm_build.py`. The tool automates the build process and provides
-assistance with installation of SDKs, running tests, etc.
+Users and developers are encouraged to use `Tools/wasm/emscripten` and
+`Tools/wasm/wasi` for building CPython for WebAssembly, as described below.
+These tools automate the build process and provide assistance with installation
+of SDKs, running tests, etc.
```

Comment:
It seems a little odd to say "below" when the instructions below say "see the devguide". Perhaps reword this to reference the dev guide for usage instructions - or maybe drop the paragraph entirely?